src/App.js

```diff
@@ -19,10 +19,11 @@ const App = () => {
                 exact={exact}
               />
             ))}
+            <Redirect to="/" />
```

Comment:
why is there a redirect method here? Normally this page is already on the landing route '/'